### PR TITLE
Let things fall behind movestones

### PR DESCRIPTION
**Pull request by Hawk777**
Fixes GH-413.

### CODE REVIEW

**Comment by numberZero** [comment]
Works for horizontal movestones, but doesn’t fix the case of a vertical one moving downwards.

File: mesecons_movestones/init.lua [hunk 1]
```diff
@@ -60,13 +60,17 @@ function mesecon.register_movestone(name, def, is_sticky, is_vertical)
...
+				mesecon.mvps_move_objects(backpos, vector.multiply(direction, -1), oldstack, -1)
+			end
 		end
-		local backpos = vector.subtract(pos, direction)
-		success, stack, oldstack = mesecon.mvps_pull_all(backpos, direction, max_pull)
-		if success then
-			mesecon.mvps_move_objects(backpos, vector.multiply(direction, -1), oldstack, -1)
+
+		-- ### Step 4: Let things fall
+		if not is_vertical then
```
Comment (numberZero) [inline mesecons_movestones/init.lua]: The check is not necessary: if it moves upwards, or is sticky, there is nothing to fall.

**Comment by numberZero** [comment]
Nvm, fixed myself.

**Comment by DS-Minetest** [comment]
:thinking: Related: #371

**Comment by DS-Minetest** [comment]
Sorry for confusing and for wasting others time.
I just thought if it would be better to do this check here for all mvps moves. But now I saw that the movestone is not moved per mvps.

**Pull request merged**